### DISCUSSION TITLE
The thermal conductivity parameters calculated according to the example Si do not accord with the practice

### DISCUSSION THREAD

**Discussion by xiexiaoyong2000** [nakib/elphbolt#167] [Q&A]
Description:
After successfully installing elphbolt, I used elphbolt to calculate the example Si you gave in the file and compared it to the calculation reference results given. The Si thermal conductivity I calculated is only 2.85W/mk, which is quite different from the reference result you gave of 86.3W/mk. I tried to modify some input parameters, but still didn't get a good solution. The relevant calculation files are attached.
Do you have any ideas on how to solve this problem?
[input.log](https://github.com/user-attachments/files/18048151/input.log)
[output.log](https://github.com/user-attachments/files/18048152/output.log)

**Comment by nakib** [comment]
Hi xiexiaoyang2000,

Comparing your output log to the reference output log, it is clear that the
third order force constants that you are using are quite different from the
ones I have. Your FORCE_CONSTANTS_3RD file has just 512 triplets, whereas
mine has 4646.

Best,
Nakib

**Reply by xiexiaoyong2000** [reply]
Hi Nakib,

Helpful! Thank you very much for your reply. I am recalculating FORCE_CONSTANTS_3RD file.

Best,
xiexiaoyang
